### PR TITLE
doc: document usage of Cloud NAT without external IP

### PR DESCRIPTION
**Pull request by tpetillot**

### CODE REVIEW

File: content/reference/admin/private_locations/configuration/gcp/index.md [hunk 1]
```diff
@@ -83,3 +94,35 @@ control-plane {
   ]
 }
 ```
+
+## Cloud NAT
```
Comment (Obero) [inline content/reference/admin/private_locations/configuration/gcp/index.md:98]:
```suggestion
## Internet access for your Load Generators instances
```

File: content/reference/admin/private_locations/configuration/gcp/index.md [hunk 2]
```diff
@@ -83,3 +94,35 @@ control-plane {
...
+
+Cloud NAT (network address translation) lets certain resources in Google Cloud create outbound connections to the internet or to other Virtual Private Cloud (VPC) networks. 
+Cloud NAT supports address translation for established inbound response packets only. **It does not allow unsolicited inbound connections.**
+
+More info on [Cloud NAT overview](https://cloud.google.com/nat/docs/overview).
+
+### Create instance without external IP
```
Comment (Obero) [inline content/reference/admin/private_locations/configuration/gcp/index.md:105]:
```suggestion
### Instances with no external IP
```

File: content/reference/admin/private_locations/configuration/gcp/index.md [hunk 3]
```diff
@@ -83,3 +94,35 @@ control-plane {
...
+
+Set the Gateway:
+- name
+- region base on your location zone
+- create a router
+
+{{< img src="cloud-nat-gateway.png" alt="Gateway configuration" >}}
+{{< img src="cloud-nat-router.png" alt="Router configuration" >}}
+
+### Static IP
```
Comment (Obero) [inline content/reference/admin/private_locations/configuration/gcp/index.md:120]:
```suggestion
### Set static IPs
```

File: content/reference/admin/private_locations/configuration/gcp/index.md [hunk 4]
```diff
@@ -83,3 +94,35 @@ control-plane {
   ]
 }
 ```
+
+## Cloud NAT
+
+Cloud NAT (network address translation) lets certain resources in Google Cloud create outbound connections to the internet or to other Virtual Private Cloud (VPC) networks.
```
Comment (Obero) [inline content/reference/admin/private_locations/configuration/gcp/index.md:100]:
```suggestion
Cloud NAT (Network Address Translation) lets certain resources in Google Cloud create outbound connections to the Internet or to an other Virtual Private Cloud (VPC) networks. 
```

File: content/reference/admin/private_locations/configuration/gcp/index.md [hunk 5]
```diff
@@ -83,3 +94,35 @@ control-plane {
...
+## Cloud NAT
+
+Cloud NAT (network address translation) lets certain resources in Google Cloud create outbound connections to the internet or to other Virtual Private Cloud (VPC) networks. 
+Cloud NAT supports address translation for established inbound response packets only. **It does not allow unsolicited inbound connections.**
+
+More info on [Cloud NAT overview](https://cloud.google.com/nat/docs/overview).
+
+### Create instance without external IP
+
+Instance without external IP cannot access internet without a Cloud NAT configured on the network, for the region.
```
Comment (Obero) [inline content/reference/admin/private_locations/configuration/gcp/index.md:107]:
```suggestion
An instance without an external IP cannot access to Internet without a Cloud NAT configured on its network, for the region.
```

**Review by Obero — APPROVED** [review]
Just some suggestions for clarity, but ok !
I'll try to test it soon before public release.

File: content/reference/admin/private_locations/configuration/gcp/index.md (hunk 5 above)
Comment (tpetillot) [reply content/reference/admin/private_locations/configuration/gcp/index.md:107]: question: why "Internet" and not "internet"?

File: content/reference/admin/private_locations/configuration/gcp/index.md (hunk 4 above)
Comment (tpetillot) [reply content/reference/admin/private_locations/configuration/gcp/index.md:100]: Idk for the end of the sentence, it was copied from google documentation 🤔

**Pull request merged**